extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -139,8 +140,7 @@ public static function onPortableInfoboxNodeImageGetData( $title, &$ref, $alt )
 		wfProfileIn( __METHOD__ );
 		if ( $title ) {
 			$details = WikiaFileHelper::getMediaDetail( $title, self::$mediaDetailConfig );
-			//TODO: When there will be more image contexts, move strings to const
-			$details['context'] = self::MEDIA_CONTEXT_INFOBOX_IMAGE;
+			$details['context'] = $details['mediaType'] === 'video' ? self::MEDIA_CONTEXT_INFOBOX_VIDEO : self::MEDIA_CONTEXT_INFOBOX_IMAGE;
```

Comment:
What would you say about extracting the context generation logic to getMediaContext($details)?
Also, assignment and ternary operator in one line benefit from some parenthesis love; While operator precedence makes this code achieve what you want, parenthesis would make recalling what that precedence is unnecessary.